.github/workflows/ci.yml

```diff
@@ -49,13 +49,13 @@ jobs:
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@v1
         # `allow-failure` not available yet https://github.com/actions/toolkit/issues/399
-        continue-on-error: ${{ matrix.version == 'nightly' }} # comment out to report nightly failures
+        #continue-on-error: ${{ matrix.version == 'nightly' }} # comment out to report nightly failures
```

Comment:
Is it the intention to keep these commented out after this is merged?

Reply:
I think it would make sense to keep them commented out until something breaks again. That way we know once things are broken again on master.

Reply:
At least for this PR we should have this commented out.

Nightly and 1.6 would have diverged a bit now though